documentation/src/docs/include/additional-modules.md

```diff
@@ -74,69 +74,87 @@ This module is part of jqwik's default dependencies.
 
```

Comment:
Show at least one example for generating dates. Compare with annotation to provide method.